src/js/containers/search/table/mockData.js

```diff
@@ -0,0 +1,28 @@
+const mockData = {
```

Comment:
The mock data can be deleted.  Though you'll need to screenshare to demo this working for QA

Reply:
will do